.github/workflows/install.yml

```
@@ -267,3 +276,9 @@ jobs:
         run: |
           is_unicorn=$(node scripts/ci/unicorn-utils.mjs is-unicorn "{\"head\": \"${NX_HEAD}\", \"base\": \"${NX_BASE}\" }")
           echo is-unicorn="$is_unicorn" | tee -a "$GITHUB_OUTPUT"
+
+      - name: DEBUG judicial affected output
+        id: judicial-debug
+        if: steps.is-judicial-affected.ouputs.AFFECTED == 'true'
```

actionlint (1.7.4)

property "ouputs" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)